tests/integration/targets/parted/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021, Alexei Znamensky
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Create empty file
+  community.general.filesize:
+    path: /bigfile
+    size: 1GiB
+  notify: Remove file
+
+- name: Create loopback device
+  command:
+    cmd: losetup -fP --show /bigfile
```

Comment:
This doesn't work on Alpine, you need to do this in two steps:
1. call `losetup -f` to get the name of the next loop device created;
2. call `losetup -f /path` to actually create it.

(See some of the other integration tests using `losetup`.)